test/overseer/core_test.clj

```diff
@@ -57,6 +57,24 @@
           graph (loom.graph/digraph {j0 []})]
       (is (not (core/valid-graph? graph))))))
 
+(def ^:private uuid-re
+  (re-pattern "[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}"))
+
+(defn- uuid-str? [s]
+  (and (string? s) (boolean (re-matches uuid-re s))))
```

Comment:
Technically the `squuid` code already goes through Java; this is really just to test that the id has been set. We could equally write `(is (:job/id job))` but I sort of liked this